src/EditorFeatures/Core/InheritanceMargin/InheritanceTargetItem.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.FindUsages;
+
+namespace Microsoft.CodeAnalysis.InheritanceMargin
+{
+    /// <summary>
+    /// Indicate the relationship between the member and its inheritance target
+    /// </summary>
+    [Flags]
+    internal enum InheritanceRelationship
+    {
+        /// <summary>
+        /// Indicate the target is implementing the member. It would be shown as I↑.
+        /// </summary>
+        Implementing = 0x1,
+
+        /// <summary>
+        /// Indicate the target is implemented by the member. It would be shown as I↓.
+        /// </summary>
+        Implemented = 0x2,
+
+        /// <summary>
+        /// Indicate the target is overriding the member. It would be shown as O↑.
+        /// </summary>
+        Overriding = 0x4,
+
+        /// <summary>
+        /// Indicate the target is overriden by the member. It would be shown as O↓.
+        /// </summary>
+        Overriden = 0x8,
```

Comment:
```suggestion
        /// Indicate the target is overridden by the member. It would be shown as O↓.
        /// </summary>
        Overridden = 0x8,
```